regression/cbmc-java/stack_var1/README

```diff
@@ -0,0 +1,2 @@
+This regression test is created from the .j file with the jasmin assembler.
+https://github.com/Sable/jasmin
```

Comment:
I added some comments about jasmin, how to get it and how to use it on `.j` files